plugins/kinet/KiNetNode.cpp

```diff
@@ -125,8 +131,8 @@ bool KiNetNode::SendDMX(const IPV4Address &target_ip, const DmxBuffer &buffer) {
 bool KiNetNode::SendPortOut(const IPV4Address &target_ip,
                             const uint8_t port,
                             const DmxBuffer &buffer) {
-  static const uint8_t flags = 0;
-  static const uint8_t padding = 0;
+  static const uint8_t flags1 = 0; // Definitely flags of some sort, seems to be impacted by buffer size
+  static const uint16_t flags2 = 0; // Possibly always 0
```

Comment:
So we've got some values for these:
https://github.com/OpenLightingProject/ola/blob/6cc16fe68d42a7edd4e273b89cf5cbbcbb530b7d/plugins/kinet/kinet.cpp#L154-L160

I've seen the 16 bit one with QPP2 at least, I'm not sure I triggered the sync one when I tried.

Reply:
The power supplies that I have will not take in 16-bit data I don't think, so I have no way to test this. Possibly I can add a virtual power supply that supports 16-bit to QuickPlay Pro 2 to get it to spit it out.

Reply:
OK, I can't get QuickPlay Pro 2 to do anything other than `0x000000` for all three flag bytes. It seems that is safe for most circumstances considering it was the same for my PDM-202 as well.

Reply:
Yeah makes sense, we've got the comments in the other file should someone ever need it and have the ability to test it.

Maybe I saw it with my other manufacturer's device then, rather than QPP2.